.github/workflows/no-response.yml

```diff
@@ -0,0 +1,23 @@
+name: No Response Bot
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # "every 6 hours"
+    - cron: '0 */6 * * *'
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
```

Comment:
Thanks for working on this! However, this action looks abandoned as well: last commit was 21 Oct 2021.

Reply:
And our current no response bot was archived on on 13 Jul 2021:
https://github.com/probot/no-response/commit/74cbc5393669059479e157084e2b3cb2ed8421c2

Reply:
The one we currently use, `probot/noresponse` is explicitly archived, so I'm 100% confident it won't be receiving future updates and that it should be replaced.

I don't think `lee-dohm/no-response` not having a commit since October 2021 means it is "abandoned". That action is very simple, so I just don't think it requires much active development.

Consider the official "checkout" action, managed directly by GitHub: https://github.com/actions/checkout/commits/main. Even that very popular action, which is used across MANY repos (https://github.com/search?l=YAML&q=%22uses%3A+actions%2Fcheckout%22&type=Code), went 4 months (November 2021 to February 2022) without any commits.

<img width="1268" alt="image" src="https://user-images.githubusercontent.com/7608904/161455891-8309d57f-1070-4552-acac-182ade9110dd.png">

Reply:
Thanks for the investigation!
I just thought that it's possible to find a more actively developing and popular action (the proposed one has only 15 stars). But I haven't managed to find the alternative.